lobby/src/main/java/org/triplea/lobby/server/db/AbstractController.java

```diff
@@ -20,4 +24,9 @@
   final Connection newDatabaseConnection() throws SQLException {
     return database.newConnection();
   }
+
+  static RuntimeException newDatabaseException(final String message, final SQLException e) {
```

Comment:
Something about "new" feels a bit weird and makes me think its actually a constructor. Might be better to use "create".